Jenkinsfile

```diff
@@ -391,6 +391,11 @@ try {
                   bat "aws s3 cp package\\win32\\build\\${packageName}-RelWithDebInfo.exe ${buildDest}/${packageName}.exe"
                   bat "aws s3 cp package\\win32\\build\\${packageName}-RelWithDebInfo.zip ${buildDest}/${packageName}.zip"
                 }
+
+                // update daily build redirect
+                withCredentials([file(credentialsId: 'www-rstudio-org-pem', variable: 'wwwRstudioOrgPem')]) {
+                  sh "docker/jenkins/publish-daily-binary.sh https://s3.amazonaws.com/rstudio-ide-build/desktop/windows/${packageName}.exe ${wwwRstudioOrgPem}"
```

Comment:
Will this work on windows, or do we need a bat equivalent?

Reply:
It's clearly been too long since I've worked with windows!   I'll update this with a .bat file.  It should be a little easier as it only has one variant to handle.

Reply:
I think it might be easier than that. This command is currently running inside the Windows container, but you can also run stuff _outside_ the Windows container. If you move this outside the `container.inside` it might just work?

Reply:
Thanks, great suggestions.  I just updated the PR... I'm not entirely sure the variables: rstudioVersionMajor, etc. will be set though where I placed the publish-daily-binary.sh script.

Reply:
Worth a shot. If it does't work out, use the "Replay" button so you can try out other approaches w/o having to submit new PRs.

Reply:
(actually, I'd be more worried about the `packageName`; you might need to `def` that earlier so it's in scope, but Groovy is largely opaque to me)